wled00/bus_manager.h

```diff
@@ -92,19 +93,21 @@ class BusDigital : public Bus {
   public:
   BusDigital(BusConfig &bc, uint8_t nr) : Bus(bc.type, bc.start) {
     if (!IS_DIGITAL(bc.type) || !bc.count) return;
-    _pins[0] = bc.pins[0];
-    if (!pinManager.allocatePin(_pins[0])) return;
-    if (IS_2PIN(bc.type)) {
-      _pins[1] = bc.pins[1];
-      if (!pinManager.allocatePin(_pins[1])) {
+    for(int i=0; i<NUM_PINS(bc.type); i++) {
+      _pins[i] = bc.pins[i];
+      Serial.print(" ");
```

Comment:
Some debug isn't bad here, but it shouldn't be plain Serial

Reply:
That's what I get from trying to submit a PR well after my brain was done for the night :-)